packages/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Link } from 'gatsby'
+import * as css from './Tabs.module.scss'
+import classnames from 'classnames'
+import { TextCaps } from '@elvia/ui'
+import {
+  HddArrowRight,
+  HddArrowLeft,
+} from '@hafslundnett/hdd-icons/hdd-icons-react'
+
+interface LinkItem {
+  name: string
+  url: string
+}
+
+export interface TabsProps {
+  className?: string
+  tabs?: LinkItem[]
+  uri?: string
+}
+
+const useWindowDimensions = () => {
+  const [width, setWidth] = React.useState(window.innerWidth)
+
+  const updateWidth = () => {
+    setWidth(window.innerWidth)
+  }
+
+  useEffect(() => {
+    window.addEventListener('resize', updateWidth)
+    return () => window.removeEventListener('resize', updateWidth)
+  })
+
+  return {
+    width,
+  }
+}
+
+export const Tabs: React.FC<TabsProps> = ({ tabs, uri }) => {
+  const [arrow, setHideArrows] = useState<string>()
+  const containerRef = useRef<HTMLElement>(null)
+
+  const hideArrowLimit = 16
+  useWindowDimensions()
+
+  const hideArrows = (element: HTMLElement | null) => {
+    if (element !== null) {
+      const scrollLeft = element.scrollLeft
+      const maxScrollLeft = element.scrollWidth - element.clientWidth
+      if (scrollLeft < hideArrowLimit) {
+        return setHideArrows('left')
+      } else if (scrollLeft >= maxScrollLeft - hideArrowLimit) {
+        return setHideArrows('right')
+      } else {
+        return setHideArrows('')
+      }
+    }
+  }
+
+  useEffect(() => {
+    hideArrows(containerRef.current) // Set state on onload
+  })
+
+  const sideScroll = (
+    element: HTMLElement | null,
+    direction: string,
+    distance: number,
+    step: number
+  ) => {
+    let scrollAmount = 0
+    function slideTimer() {
+      direction == 'left'
+        ? (element!.scrollLeft -= step)
+        : (element!.scrollLeft += step)
+      scrollAmount += step
+      if (scrollAmount < distance) {
+        requestAnimationFrame(slideTimer)
+      }
+    }
+    slideTimer()
+  }
+
+  const hasOffset = () => {
+    return containerRef.current != null &&
```

Comment:
```suggestion
    return containerRef.current !== null &&
```
trippel lik (eller trippel "ikke lik")